fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -327,6 +327,8 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     public static final String BIG_QUERY_LOG_SCAN_BYTES_THRESHOLD = "big_query_log_scan_bytes_threshold";
     public static final String BIG_QUERY_LOG_SCAN_ROWS_THRESHOLD = "big_query_log_scan_rows_threshold";
 
+    public static final String USE_PARSER_DIALECT = "use_parser_dialect";
```

Comment:
Would sql_dialect be a better name? Because we may support more dialects in the future, "starrocks" is also a kind of dialect